### DISCUSSION TITLE
Keypad light, etc shows as light after update

### DISCUSSION THREAD

**Discussion by todesto** [dgreif/ring#1034] [Q&A] [answered]
after latest update, I see base station, keypad light as powered on in Homekit.

I tried to find a way to turn it off but I don't see any.

Is this a bug in new update?

**Comment by dgreif** [comment, marked as answer]
Just like every device in the plugin, you can hide these using the `hideDeviceIds` option. The id for each device is logged when homebridge starts. My default is to show _everything_ and let users hide the things they don't wan't/care about.

**Reply by Mr-Bridge-0** [reply]
Trying to turn use the HideDeviceIDs as well, but can't figure out the he exact Device ID. Is it the full 32 digits that I find in the startup logs next to the device name? Can you share an example?

**Reply by dgreif** [reply]
Yes, it's the ID in the startup logs

**Reply by todesto** [reply]
you need to get the device ID and add in optional configuration in setting.
<img width="757" alt="Screen Shot 2022-09-17 at 8 47 19 PM" src="https://user-images.githubusercontent.com/11996603/190884829-680b300c-5d29-46bc-83fb-177dc75c97eb.png">

**Reply by Mr-Bridge-0** [reply]
Got it to work, thanks guys, I had a typo on my first try haha
